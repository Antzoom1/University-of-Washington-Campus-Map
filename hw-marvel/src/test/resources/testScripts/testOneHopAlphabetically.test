# Tests the BFS on the staffSuperheroes csv file and adds nodes and edges to ensure
# that a path with one hop can be found

LoadGraph staffSuperheroes staffSuperheroes.csv

AddNode staffSuperheroes OnePunchMan
AddEdge staffSuperheroes Ernst-the-Bicycling-Wizard OnePunchMan CSE331
AddEdge staffSuperheroes OnePunchMan Ernst-the-Bicycling-Wizard CSE331

FindPath staffSuperheroes OnePunchMan Ernst-the-Bicycling-Wizard