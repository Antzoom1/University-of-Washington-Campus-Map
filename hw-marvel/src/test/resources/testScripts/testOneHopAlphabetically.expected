# Tests the BFS on the staffSuperheroes csv file and adds nodes and edges to ensure
# that a path with one hop can be found

loaded graph staffSuperheroes

added node OnePunchMan to staffSuperheroes
added edge CSE331 from Ernst-the-Bicycling-Wizard to OnePunchMan in staffSuperheroes
added edge CSE331 from OnePunchMan to Ernst-the-Bicycling-Wizard in staffSuperheroes

path from OnePunchMan to Ernst-the-Bicycling-Wizard:
OnePunchMan to Ernst-the-Bicycling-Wizard via CSE331